tutorials/tools/Text_Normalization_Tutorial.ipynb

```diff
@@ -0,0 +1,472 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "1C9DdMfvRFM-"
+      },
+      "source": [
+        "print(TOOLS_DIR)\n",
+        "! ls -l $TOOLS_DIR\n",
+        "! ls -l $TOOLS_DATA_DIR"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "XUEncnqTIzF6"
+      },
+      "source": [
+        "# Data Preparation and Download\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "YDVtdJxhZWju"
+      },
+      "source": [
+        "## Data for Prediction\r\n",
+        "For prediction, let's download a text file from [http://www.gutenberg.org/files/48874/48874-0.txt](http://www.gutenberg.org/files/48874/48874-0.txt)."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "bkeKX2I_tIgV"
+      },
+      "source": [
+        "## create data directory and download an audio file\n",
+        "WORK_DIR = 'WORK_DIR'\n",
+        "DATA_DIR = WORK_DIR + '/DATA'\n",
+        "os.makedirs(DATA_DIR, exist_ok=True)\n",
+        "text_file = '48874-0.txt'\n",
+        "if not os.path.exists(os.path.join(DATA_DIR, text_file)):\n",
+        "    print('Downloading text file')\n",
+        "    wget.download('http://www.gutenberg.org/files/48874/' + text_file, DATA_DIR)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "yyUE_t4vw2et"
+      },
+      "source": [
+        "The `DATA_DIR` should now contain the text file"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "VXrTzTyIpzE8"
+      },
+      "source": [
+        "!ls -l $DATA_DIR"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "FWqlbSryw_WL"
+      },
+      "source": [
+        "print the first 10 lines of the file :"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "1vC2DHawIGt8"
+      },
+      "source": [
+        "! head -n 10 $DATA_DIR/$text_file"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "RMT5lkPYzZHK"
+      },
+      "source": [
+        "## Data for Evaluation\n",
+        "\n",
+        "The data for Evaluation needs to be segmented and labeled by semiotic class, following the format of [Google Text normalization dataset](https://www.kaggle.com/richardwilliamsproat/text-normalization-for-english-russian-and-polish).\n",
+        "That is, every line of the file needs to have the format <semiotic class>\\t<unnormalized text>\\t<`self` if trivial class or normalized text>.\n",
```

Comment:
What is \t\t in \t\t<self..? the formatting might be off

Reply:
fixed